app/views/countries/partials/_protection-dials.erb

```diff
@@ -0,0 +1,51 @@
+<section class="container section-padding--bottom">
+  <div>
+    <h2>
+      <%= t('countries.shared.protection_dials.title_html'); %>
+      <modal-trigger id="citation" :modal-content="<%= modal_content %>" class="icon--citation margin-space-left"></modal-trigger>
+    </h2>
+
+    <% t('countries.shared.protection_dials.description_html').each do |p| %>
+      <p><%= p.html_safe %></p>
+    <% end %>
+  </div>
+
+  <div class="cards--protection-dials">
+    <% map_datasets.each do |dataset| %>
+      <%
+        dataset_status = @country.occurrences[dataset[:id]]
+        dataset_status_title = I18n.t("countries.shared.habitats_present.title_#{@country.occurrences[dataset[:id]]}")
+        icon_class = get_habitat_icon_class(dataset[:id], @country.occurrences[dataset[:id]])
+        protected_percentage = @country.protection_stats[dataset[:id]] ? @country.protection_stats[dataset[:id]]['protected_percentage'].round(2) : 0
+        total_value = @country.protection_stats[dataset[:id]] ? @country.protection_stats[dataset[:id]]['total_value'].to_i : 0
+        total_units = dataset[:id] == 'coldcorals' ? 'observations' : 'km<sup>2</sup>'.html_safe
```

Comment:
I would avoid assigning variables in the view as much as possible. So perhaps those could be moved into the `countries_helper` which already exists in `app/helpers/`.

Some of those could also be reused like embedding the `dataset_status` within the `dataset_status_title`